src/daemon/demo.sh

```diff
@@ -5,8 +5,15 @@ unset "DAEMON_OPTS[${#DAEMON_OPTS[@]}-1]" # remove the last element of the array
 : "${OSD_PATH:=/var/lib/ceph/osd/${CLUSTER}-0}"
 PREPARE_OSD=$OSD_PATH
 ACTIVATE_OSD=$OSD_PATH
-MDS_PATH="/var/lib/ceph/mds/${CLUSTER}-0"
+# the following ceph version can start with a numerical value where the new ones need a proper name
+if [[ "$CEPH_VERSION" == "jewel" || "$CEPH_VERSION" == "kraken" || "$CEPH_VERSION" == "luminous" ]]; then
+  MDS_NAME=0
+else
+  MDS_NAME=demo
+fi
+MDS_PATH="/var/lib/ceph/mds/${CLUSTER}-$MDS_NAME"
 RGW_PATH="/var/lib/ceph/radosgw/${CLUSTER}-rgw.${RGW_NAME}"
+# shellcheck disable=SC2153
```

Comment:
I don't get why you need a SC2153 here.

Reply:
shellcheck believes that MGR should be MDS because MDS is defined and not MGR. But actually MGR is sourced from another file so shellcheck does not know it.

Reply:
thx.